services/auth/src/signaturevalidator.js

```diff
@@ -349,20 +349,6 @@ const createSignatureValidator = function(options) {
         message: message.toString(),
       };
     }
-    options.monitor.log.signatureValidation({
```

Comment:
I think we might want to still logs this, just without `scopes`?

Reply:
The authenticateHawk call will be logged.. is that enough?  What is this useful for?

Reply:
I ask because this is the third log line per API method call, so it's high-volume..

Reply:
Ah, good point. I guess if we make sure that all of the contents of this message end up in the `apiMethod` message then we can remove this.

The only other consideration is that we know this one will end up in papertrail. Not sure if the data that is in `authenticateHawk`'s `apiMethod` message is enough to replace this?

In [bug 1538010](https://bugzilla.mozilla.org/show_bug.cgi?id=1538010) we have started sending this off to security's setup. We should let them know this is changing or going away depending on what we decide to do.

Reply:
I'll leave this in for now, and filed [bug 1544942](https://bugzilla.mozilla.org/show_bug.cgi?id=1544942) to turn it off.